README.md

```diff
@@ -154,6 +154,22 @@ kubectl delete namespace kogito-workflows
 make undeploy
 ```
 
+## Use Workflow Development Mode
+
+#### Override Builder image and version
+
+By default the builder image use a stable version aligned to the Kogito version.
+
+In the dev mode you can be override the default builder image using the devBaseImage in the KogitoServerlessPlatform:
+
+```sh
```

Comment:
Isn't better to add a full KogitoServerlessPlatform example?

Reply:
ok